backend/cli/parser/__init__.py

```diff
@@ -53,7 +53,7 @@ def correctness_commands(params):
 
     else:
         print("""Input command!:
-        ./main.py [-p][--path] <command> <arguments>
+        ./manage.py [-p][--path] <command> <arguments>
```

Comment:
./main.py [-p][--path] <command> <arguments>
```suggestion
        ./manage.py [-p][--path] <command> <arguments>
```